cfpb/consumerfinance.gov
# Paying for College - Fix fatal errors in schoolModel

This PR will fix some fatal errors that occur when schoolData in our API has unexpected `null` values.

## Additions
- Add checks for potential null values in school data from API endpoint

## How to test this PR
Some aspects of this PR are hard to test. Using "University of Phoenix - Phoenix Campus" as the school will result in a `null` for `programsPopular` (which should no longer cause a fatal error). I also added fixes in case `state` or `region` are `null`, since that could also cause a fatal error in the API call.

1. Pull down the branch and get it running!
2. Use the "University of Phoenix - Phoenix Campus" for your school and hopefully the app will not encounter a fatal error!

## Checklist
